src/_imagingft.c

```diff
@@ -1204,6 +1205,16 @@ font_getvarnames(FontObject *self) {
 
     num_namedstyles = master->num_namedstyles;
     list_names = PyList_New(num_namedstyles);
+
+    int *list_names_filled = PyMem_Malloc(num_namedstyles * sizeof(int));
```

Comment:
This was my bad, but there should be a `PyMem_Free` at the end of the function for this. Sorry!